src/main/resources/application.properties

```diff
@@ -160,18 +160,18 @@ logging.logback.rollingpolicy.max-file-size=20KB
 logging.logback.rollingpolicy.max-history=1
 
 ### SMS Gateway URL details
-sms-username=<Enter SMS username>
-sms-password=<Enter SMS password>
-send-message-url=<Enter SMS sending URL here>
+sms-username=PIRAMAL_SW_AHxoyCXejeJba13oKHcv
```

Comment:
@helenKaryamsetty We should not keep any sensitive information like URL and credentials in public repository. Please revert this to the previous values. The values can be updated directly in war file during deployment.